io.github.stella_emu.Stella.yaml

```diff
@@ -0,0 +1,32 @@
+app-id: io.github.stella_emu.Stella
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+
+rename-desktop-file: stella.desktop
+rename-icon: stella
+
+cleanup:
+  - /lib
+  - /share/doc
+
+finish-args:
+  - --filesystem=home:rw
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=all
+
+modules:
+  - name: stella
+    sources:
```

Comment:
Typically, `sources` are the last element of a module. It doesn't change anything, just a style remark.